tests/integration/DynamicSegments/Filters/EmailActionTest.php

```diff
@@ -45,49 +45,49 @@ public function _before() {
 
   public function testGetOpened() {
     $emailAction = new EmailAction(EmailAction::ACTION_OPENED, $this->newsletter->id);
-    $sql = $emailAction->toSql(Subscriber::selectExpr('*'));
+    $sql = $emailAction->toSql(Subscriber::selectExpr('*')); // @phpstan-ignore-line
```

Comment:
There is an error:
`Parameter #1 $orm of method MailPoet\DynamicSegments\Filters\EmailAction::toSql() expects MailPoetVendor\Idiorm\ORM, MailPoet\Models\Subscriber given.`
And haven't found the solution. Because it's an old model I didn't spend much time with this.